gamedata/descriptions/levels/shop.md

```diff
@@ -0,0 +1,6 @@
+Сan you get the item from the shop for less than the price asked?
+This elevator won't let you reach the top of your building. Right?
```

Comment:
Don't understand the part about the elevator. Is this a typo from another level?

Reply:
Yes, just typo.

Reply:
Pls fix